noccur.el

```diff
@@ -54,13 +54,26 @@ ls-files' and 'grep'."
   (interactive (occur-read-primary-args))
   (let* ((default-directory (read-directory-name "Search in directory: "))
          (files (mapcar #'find-file-noselect
-                        (noccur--find-files regexp))))
+                        (noccur--find-files-project regexp))))
+    (multi-occur files regexp nlines)))
+
+;;;###autoload
+(defun noccur-directory (regexp &optional nlines)
```

Comment:
I find the name confusing, as it's IMO unclear that the search won't be recursive.

What about the following: `noccur-directory` could be recursive by default unless called with a prefix argument?